exec/java-exec/src/test/java/org/apache/drill/PlanTestBase.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.drill;
 
+import java.nio.file.Paths;
```

Comment:
We use the Hadoop library's  `Path` class everywhere .. is that not sufficient for your use ?

Reply:
Looks like we can't use Hadoop library path here and moreover this test imports java.nio.file.Files and there are a lot of tests that import java.nio.file.Paths.